flink-end-to-end-tests/test-scripts/test-runner-common.sh

```diff
@@ -46,7 +46,7 @@ function run_test {
 
     function test_error() {
       echo "[FAIL] Test script contains errors."
-      post_test_validation 1 "$description" "$skip_check_exceptions"
+      post_test_validation 1 "$description"
```

Comment:
do you know whether the bash tests upload the logs for failed tests?

Reply:
My impressions is that this will actually obfuscate errors. People pretty much only look at the last ~20 lines of the CI output, and don't properly investigate the case, resulting in several superfluous JIRAs.

Reply:
> do you know whether the bash tests upload the logs for failed tests?

No. All directories are cleaned up after a test (in any case).

> My impressions is that this will actually obfuscate errors. People pretty much only look at the last ~20 lines of the CI output, and don't properly investigate the case, resulting in several superfluous JIRAs.

That is definitely a risk, I agree. But I would prefer not to rework this change for a problem we have not confirmed to exist.
It is quite tedious to improve these things because of all the testing cycles in between.
I would prefer to add this as is and re-adjust later if we have the feeling that people are not checking the logs carefully enough.

Reply:
> But I would prefer not to rework this change for a problem we have not confirmed to exist.

We have repeatedly seen people not properly analyzing the CI output when the logs are being dumped, this is not a theory.

> It is quite tedious to improve these things because of all the testing cycles in between.

That comes with the territory unfortunately. You will just have to find a was to reduce these cycles, e.g., only compile and test flink-core or refactor the scripts such that you can run things offline.

Reply:
Okay. I will look into a different approach for making the logs available.